src/libraries/System.Private.CoreLib/src/System/Reflection/MethodInfo.cs

```diff
@@ -26,6 +26,12 @@ public abstract partial class MethodInfo : MethodBase
         public virtual Delegate CreateDelegate(Type delegateType) { throw new NotSupportedException(SR.NotSupported_SubclassOverride); }
         public virtual Delegate CreateDelegate(Type delegateType, object? target) { throw new NotSupportedException(SR.NotSupported_SubclassOverride); }
 
+        /// <summary>Creates a delegate of the given type 'T' from this method.</summary>
+        public T CreateDelegate<T>() where T : Delegate => (T)CreateDelegate(typeof(T));
```

Comment:
```suggestion
        public T CreateDelegate<T>() where T : System.Delegate => (T)CreateDelegate(typeof(T));
```

I assume that it is what you would get if you regenerated the file using https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md

Reply:
Since this one is the implementation, not the ref, I think the comment got misapplied :)

Reply:
Yes, you are right. This comment should be under ref/...